edgelet/iotedge/src/lib.rs

```diff
@@ -23,6 +23,7 @@ mod check;
 mod client;
 pub mod config;
 mod error;
+mod internal;
```

Comment:
why define in internal and not in config folder directly?

Reply:
Because we dont want to expose this to outside components.